torchdrivesim/rendering/cv2.py

```diff
@@ -25,6 +26,16 @@ def __init__(self, cfg: CV2RendererConfig, *args, **kwargs):
 
     def render_mesh(self, mesh: BirdviewMesh, res: Resolution, cameras: Cameras) -> torch.Tensor:
 
+        if self.cfg.trim_mesh_before_rendering:
+            # For efficiency, remove faces that are not visible anyway
+            viewing_polygon = cameras.reverse_transform_points_screen(
+                torch.tensor([
+                    [0, 0], [0, res.height], [res.width, res.height], [res.width, 0]
+                ], device=mesh.device), res=res
+            )
+            viewing_polygon = viewing_polygon * 1.05  # safety margin
```

Comment:
Don’t we need to  minus center before apply the scale ?

Reply:
Yes, thanks for catching that.